google-cloud-logging/src/test/java/com/google/cloud/logging/LoggingImplTest.java

```diff
@@ -1852,7 +1894,8 @@ public void testWriteLogEntriesAsyncWithOptions() {
         ImmutableList.of(LOG_ENTRY1, LOG_ENTRY2),
         WriteOption.logName(LOG_NAME),
         WriteOption.resource(MONITORED_RESOURCE),
-        WriteOption.labels(labels));
+        WriteOption.labels(labels),
+        WriteOption.destination(LogDestinationName.project(PROJECT)));
```

Comment:
Do we have a test that validates the behavior of the `write()` **without** destination WriteOption?

Reply:
Perhaps `testWriteLogEntriesAsync()` answers your question?

Reply:
I think it is not. I was looking for a test that validating the "previous" logic when a user provides log name option and does not provide destination.

Reply:
Sure - adding missing part